src/vs/workbench/parts/localizations/electron-browser/localizations.contribution.ts

```diff
@@ -141,19 +141,15 @@ export class LocalizationWorkbenchContribution extends Disposable implements IWo
 					return;
 				}
 
-				const extensionIdPostfix = this.getPossibleChineseMapping(locale);
-				const ceintlExtensionSearch = this.galleryService.query({ names: [`MS-CEINTL.vscode-language-pack-${extensionIdPostfix}`], pageSize: 1 });
-				const tagSearch = this.galleryService.query({ text: `tag:lp-${locale}`, pageSize: 1 });
-
-				TPromise.join([ceintlExtensionSearch, tagSearch]).then(([ceintlResult, tagResult]) => {
-					if (ceintlResult.total === 0 && tagResult.total === 0) {
+				this.galleryService.query({ text: `tag:lp-${locale}`, pageSize: 1 }).then(tagResult => {
```

Comment:
Since page size is 1, we might not get all extensions in first page. Since we are looking extensions only in first page, this could miss MS-CEINTL extension